### PR TITLE
Added IO metrics

### PR DESCRIPTION
**Pull request by Zyqsempai**
Added IO metrics.
Signed-off-by: Boris Popovschi <zyqsempai@mail.ru>

### CODE REVIEW

**Comment by Zyqsempai** [comment]
@estesp @AkihiroSuda PTAL

**Comment by crosbymichael** [comment]
@Zyqsempai shouldn't this be keyed off of major:minor that you get from reading the cgroup files?

File: v2/stats/metrics.proto [hunk 1]
```diff
@@ -74,5 +75,15 @@ message RdmaEntry {
...
+}
+
+message IOEntry {
+	uint64 major = 1;
+	uint64 minor = 2;
+	uint64 rbytes = 3;
+	uint64 wbytes = 4;
+	uint64 rios = 5;
+	uint64 wios = 6;
+}
```
Comment (fuweid) [inline v2/stats/metrics.proto]: unexpected eof. :)

Comment (Zyqsempai) [reply v2/stats/metrics.proto]: ufff;) that was hard but i fixed it

**Comment by Zyqsempai** [comment]
@AkihiroSuda @crosbymichael PTAL

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**